lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -364,7 +366,8 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
             const parentWidgetId = this.linkedWidgetId;
             bodyParam[parentWidgetId] = parentWidgetValue;
         }
-        return bodyParam;
+
+        return this.formUtilsService.getRestUrlVariablesMap(this.field.form, this.field.restUrl, bodyParam);
```

Comment:
So the backend expects a flat body object where the variables and the parentWidgetId are mixed? Theoretically, this could cause a collision. Would it be an overkill to check this?

Reply:
I could not find the meaning of the parent widget ID in getRestWidgetData. If I understand the usage correctly, it is also resolved in the rest URL as a variable. Maybe it is actually unused. I also wonder why the parent widget ID has a meaning in a dropdown, but not in the radio button. Not sure how we could get insights into this.

Reply:
There are also no existing Unit Tests verifying the existence of this parameter in the body, so we might can safely remove it ;)

Reply:
Okay seems like there might be a parameter in the URL: https://hyland.atlassian.net/browse/AAE-6025